### PR TITLE
Change terminology for allowed and disallowed entities

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
This touches some old / archived policies, but hey things sometimes need to be changed. We can discuss what the best new terminology should be if that's useful. But I took a first-pass at what I thought made sense.

## Issue / Bugzilla link
None

## Testing
- [ ] Did I miss any other references?

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/plugin-permission-policy.html [hunk 1]
```diff
@@ -106,8 +106,8 @@ <h2>{{ _('Version 1.0') }}</h2>
 
 <p>
     {% trans
-        whitelist_email='href=mailto:pluginwhitelist@mozilla.com' %}
-    Please contact Mozilla at <a {{ whitelist_email }}>pluginwhitelist@mozilla.com</a> for more information about this policy and answers to related questions.
+        allowedlist_email='href=mailto:pluginwhitelist@mozilla.com' %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/plugin-permission-policy.html:109]: Note: I'm not able to change this email address, but perhaps we can find where this goes to and issue a new / different address?

**Comment by alexgibson** [comment]
Adding a do-not-merge until we run these changes by folks.

**Comment by pmac** [comment]
Oh wow! That is a lot. Great work. +1 from me.

**Comment by hoosteeno** [comment]
A [conversation in #firefox](https://mozilla.slack.com/archives/C4D3JFF26/p1591973257121400) confirms that we can simply EOL and 301 two of the documents in this PR to archives on wikimo. 

1. 301 https://www.mozilla.org/about/governance/policies/security/plugin-whitelist-policy/ -> https://wiki.mozilla.org/Plugins/Firefox_Whitelist
2. 301 https://www.mozilla.org/about/governance/policies/security-group/tld-idn/ -> https://wiki.mozilla.org/IDN_Display_Algorithm

**Comment by alexgibson** [comment]
Thanks @hoosteeno, I've removed and redirected the pages as suggested.

I think this is now ready for code review and can be merged. @pmac r?

**Comment by alexgibson** [comment]
Ok, waiting on some legal feedback before we merge.

**Comment by ejregithub** [comment]
@alexgibson @pmac @hoosteeno - per MFeldman's response to Hoosteeno's email, it's fine to make these changes. 

We're good to go here.

**Pull request merged**